frontend/src/types.ts

```diff
@@ -3020,6 +3020,20 @@ export interface ExportedAssetType {
     expires_after?: Dayjs
 }
 
+export type QueryStatus = {
+    id: string
+    async: boolean
+    team_id: number
+    error: boolean
+    complete: boolean
+    error_message: string
+    results: any
+    start_time?: string
+    end_time?: string
+    expiration_time?: string
+    task_id?: string
+}
```

Comment:
Perhaps this could go into `schema.ts` instead of `types.ts`? Then running `pnpm run schema:build` will automatically generate a copy of this in `schema.py`, and it can be used instead of the dataclass in posthog/clickhouse/client/execute_async.py?

Reply:
Ah that's good to know. Will check out how this works 👍🏻